src/EnergyPlus/DataEnvironment.hh

```diff
@@ -22,6 +22,7 @@ namespace DataEnvironment {
 	extern Real64 const EarthRadius; // Radius of the Earth (m)
 	extern Real64 const AtmosphericTempGradient; // Standard atmospheric air temperature gradient (K/m)
 	extern Real64 const SunIsUpValue; // if Cos Zenith Angle of the sun is >= this value, the sun is "up"
+	extern Real64 const StdBaroPressSeaLevel; // Standard barometric pressure at sea level (Pa)
```

Comment:
@EnergyArchmage I would prefer "sea level" in the name to help clarify.  "std" and "STP" are too ambiguous here.  The comments are clear, but that gets lost when used elsewhere in the code.